app/controllers/application_controller.rb

```diff
@@ -1,6 +1,14 @@
 class ApplicationController < ActionController::Base
   before_action :set_locale
 
+  private
+  rescue_from ActiveRecord::RecordNotFound, with: :handle_record_not_found
```

Comment:
cho xuống private có chạy dc k hầy

Reply:
em vẫn chạy được và in ra flash báo lỗi ạ